src/app/helptext/services/components/service-ftp.ts

```diff
@@ -93,21 +94,23 @@ passiveportsmax_tooltip: T('Used by clients in PASV mode. A default of <i>0</i>\
  means any port above 1023.'),
 passiveportsmax_validation: [rangeValidator(0, 65535), Validators.required],
 
-localuserbw_placeholder : T('Local User Upload Bandwidth'),
-localuserbw_tooltip: T('In KiB/s. A default of <i>0</i> means unlimited.'),
-localuserbw_validation: [rangeValidator(0), Validators.required],
+localuserbw_placeholder : T(`Local User Upload Bandwidth: ${globalHelptext.human_readable.suggestion_label}`),
```

Comment:
the translation extraction does not work with variables, you'll need to do translate.get on these things both separately and concatenate them